test/service_stubs/heroku_api_stub.rb

```diff
@@ -50,8 +50,9 @@ def two_factor?
     MultiJson.encode({ email: "kerry@heroku.com" })
   end
 
-  post "/confirm_change_email/:token" do |token|
-    302
+  patch "/users/~" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.